museek/antenna_sanity/from_log.py

```diff
@@ -0,0 +1,24 @@
+class FromLog:
+    def __init__(self, obs_script_log: list):
+        self.obs_script_log = obs_script_log
+
+    def straggler_list(self) -> list[str]:
```

Comment:
whenever anyone wants to have the `straggler_list`, the method will be run again.

What we can do:
- make this method private
- create a new method with the name `get_straggler_list()`: check if `self.straggler_list` is not None, and then either just return that or run `self._straggler_list()` and store that to `self.straggler_list`.